.travis.yml

```diff
@@ -1,34 +1,13 @@
+dist: xenial
+sudo: false
 language: python
-cache:
-  pip: true
 addons:
-  postgresql: 9.6
+  postgresql: "9.6"
```

Comment:
might be worthwhile to also upgrade postgresql too?

Reply:
I think 9.6 is xenial default, and we only need very little Postgres, so I' leave it like this for now.